roles/aodh/templates/etc/serverspec/aodh_spec.rb

```diff
@@ -33,6 +28,18 @@
   it { should be_owned_by 'aodh' }
 end
 
+{% if inventory_hostname in groups['controller'] %}
+processes = ['aodh-api',
+  'aodh-evaluator',
+  'aodh-listener',
+  'aodh-notifier']
+processes.each do |process|
+  describe process(process) do
+    its(:user) { should eq 'aodh' }
```

Comment:
I think we're also checking the process in mysql, rabbit, and heat roles. So if this is wrong, we should change them all.